Documents/Management/WeeklyReports/weeklyReportThirdWeek.md

```diff
@@ -41,7 +41,7 @@ The difficulties encountered during this week were mainly due to the hardware co
    - Continue the user manual.
 
 2. Hardware components
-   - The team will have a new battery to test the project's functionality.
+   - The team will have a new battery to test the project's functionality, This new battery was ordered last week by Mathias because we thought the battery was necessary to test the project.
```

Comment:
After a comma we don't have to use a uppercase.

Reply:
True.